static/sass/styles-v1.scss

```diff
@@ -12,6 +12,13 @@
 // vendor
 @import 'vendors/global-nav/build';
 
+// patterns
```

Comment:
What confuses me is that both files are called `styles-v1.scss` just ones in a folder called css and the other in a sass folder.